.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: "❗️ Bug report"
+about: Create a report to help us improve
+labels: kind/bug
+---
+
+> Bugs should be filed for issues encountered whilst operating Tekton Dashboard.
+> You should first attempt to resolve your issues through the community support channels,
+> e.g. [Slack](https://app.slack.com/client/TJ45YV83X/CHTHGRQBC).
```

Comment:
How about linking to this https://github.com/tektoncd/community/blob/master/contact.md#slack and then suggesting the dashboard channel? They can find instructions on how to get an invite link here

Reply:
I agree, sounds a lot better. Fixed